code/script.js

```diff
@@ -218,71 +229,135 @@ const generateBoard = () => {
           <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
         </div>
       </div>
-    `
-  })
-}
+    `;
+  });
+};
 
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-}
+  secret =
+    charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)];
+};
 
 // This function to start (and restart) the game
 const start = () => {
+  onGame ? location.reload() : (onGame = true);
+  score = MAX_ATTEMP;
+  updateScore();
```

Comment:
I like this updateScore funcion! I'm going to try it myself next time - thank you!